.gitignore

```diff
@@ -0,0 +1 @@
+secrethub
```

Comment:
I think this will match (recursively) all files/dirs named 'secrethub' not just the one on the `.gitignore` level. We should add a `/` to the beginning of the line to only match files named secrethub that are next to the `.gitignore`.

> If there is a separator at the beginning or middle (or both) of the pattern, then the pattern is relative to the directory level of the particular .gitignore file itself. Otherwise the pattern may also match at any level below the .gitignore level.

Source: https://git-scm.com/docs/gitignore

Reply:
Good spot!